simian/render.py

```diff
@@ -94,7 +109,13 @@ def render_scene(
 
     os.makedirs(output_dir, exist_ok=True)
 
-    initialize_scene()
+    if user_blend_file:
```

Comment:
this seems wrong, like, we want to init the scene either, way, right? we gotta create camera rig etc

Reply:
agreed